src/download_and_link.rs

```diff
@@ -65,6 +65,31 @@ impl DownloadAndLinkHooks for () {
 	type Error = Infallible;
 }
 
+/// Options for which dependencies to install.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Default)]
+pub enum InstallDependenciesMode {
+	/// Install all dependencies.
+	#[default]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
pub enum InstallDependenciesMode {
	/// Install all dependencies.
```
Let's not implement more traits than useful.